packages/formation-react/src/components/Table/Table.jsx

```diff
@@ -48,19 +49,26 @@ function Table(props) {
             className={`${borderClasses} ${rowPaddingClass}`}
             role="row"
           >
-            {fields.map((field, index) => (
-              <td
-                data-index={index}
-                className={classNames(borderClasses, {
-                  'vads-u-text-align--left': field.alignLeft,
-                })}
-                data-label={`${field.label}:`}
-                key={`${rowIndex}-${field.label}`}
-                role="cell"
-              >
-                {item[field.value] === null ? '---' : item[field.value]}
-              </td>
-            ))}
+            {fields.map((field, index) => {
+              if (field.alignRight) {
+                alignClass = 'medium-screen:vads-u-text-align--right';
+              } else if (field.alignLight) {
+                alignClass = 'vads-u-text-align--left';
+              } else {
+                alignClass = 'vads-u-text-align--left';
```

Comment:
This `else` does the same thing as the `else if` above it. If my previous comment isn't feasible to implement, then we should change how this if statement is set up.

Reply:
@bkjohnson Yep, I updated this to be inline using `classNames`